src/main/java/com/dmgrt/cinema/models/MovieSession.java

```diff
@@ -0,0 +1,60 @@
+package com.dmgrt.cinema.models;
+
+import java.time.LocalDateTime;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "movie_sessions")
+public class MovieSession {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne
+    private Movie movie;
+    @ManyToOne
```

Comment:
this is for your information:  keep in mind that toOne relations have fetchType eager by default, which is not always quite goo

Reply:
thanks